### ISSUE TITLE
Add the ability to configure a grid mapfile for origins

### ISSUE THREAD

**Issue by jhiemstrawisc** [PelicanPlatform/pelican#1419]
Pelican Service:
- [ ] Client
- [ ] Plugin
- [ ] Registry
- [ ] Director
- [x] Origin
- [ ] Cache
- [ ] Other (please give the detail)

**Is your feature request related to a problem? Please describe.**
Some legacy services cannot be migrated to Pelican without the ability to specify/configure a grid mapfile for xrootd.

**Describe the solution you'd like**
Add a new configuration param, `Xrootd.GridMapfile` that, when set, adds the following configuration to the xrootd config:
```
sec.protocol gsi -gmapopt:trymap -gmapto:0 -gridmap:/path/to/mapfile
```

**Comment by jhiemstrawisc** [comment]
@bbockelm, I think this is one of those issues I wrote up after a discussion with you, and I only half understood the premise then (and don't understand what it's for now).

Is this one to stick in parking lot, make a priority in 7.13/14 or to close as won't fix?

**Comment by bbockelm** [comment]
I think we need to keep this around -- I need to talk to LIGO folks to understand if there's a workaround for this or not (if there's no workaround, we'll need to implement the functionality before we can upgrade them to Pelican).

**Comment by bbockelm** [comment]
I think we've found various workarounds subsequently.  Closing as not planned.
